spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -38,6 +57,106 @@
           expect(cur_plat._platform.target_user).to eq(plat[:target_user])
         end
       end
+
+      describe '#generate_msi_packaging_artifacts' do
+        before do
```

Comment:
I'd suggest always calling out `before :each` or `before :all` explicitly. Life's to short to memorize default values.